bleak/backends/winrt/client.py

```diff
@@ -152,12 +157,17 @@ async def connect(self, **kwargs) -> bool:
 
         Keyword Args:
             timeout (float): Timeout for required ``BleakScanner.find_device_by_address`` call. Defaults to 10.0.
+            use_cached_services (bool): ``True`` allows Windows to fetch the services, characteristics and descriptors
```

Comment:
Adding the kwarg to the connect method too seems a bit much. What is the use case?

Reply:
It much, I agree. But it is what initiates the `get_services` in the non-context manager way of connecting, and it felt appropriate to add it there as well if it was also in `get_services`. Maybe I should remove it from both and just go with the constructor...

Reply:
We could leave them out for now and we can always add them later if there is a real use case for it.

Reply:
Removed it from connect. It is not needed there.